### ISSUE TITLE
CR didn't work

### ISSUE THREAD

**Issue by zhaorendong** [kiali/kiali#1908]
I created Kiali on kubernetes and it worked  all right, the config map of Kiali set the strategy "login".Than I created the Operator and CR, but the  strategy that I set it "anonymous" not worked.
Why does this happen? Did I configure it right？
Kiali config map:
apiVersion: v1
kind: ConfigMap
metadata:
  name: kiali
  namespace: istio-system
  labels:
    app: kiali
    chart: kiali
    heritage: Tiller
    release: istio
data:
  config.yaml: |
    istio_namespace: istio-system
    auth:
      strategy: login
    server:
      port: 20001
      web_root: /kiali
    external_services:
      tracing:
        url: 
      grafana:
        url: 
      prometheus:
        url: http://prometheus:9090
CR:
apiVersion: kiali.io/v1alpha1
kind: Kiali
metadata:
  name: kiali
spec:
  auth:
    strategy: anonymous
  external_services: {}
  grafana: {}
  istio_namespace: istio-system
  prometheus:
    url: http://prometheus:9090
  server:
    port: 20001
    web_root: /kiali

**Comment by lucasponce** [comment]
I've added @jmazzitelli to review it.

**Comment by jmazzitelli** [comment]
This part confuses me:

>I created Kiali on kubernetes and it worked all right,
> the config map of Kiali set the strategy "login".Than I created the Operator and CR

Did you installed Kiali first and then deployed the operator and created a CR? That is incorrect. You do not install Kiali first - you install the Operator, then you create a CR, and then the operator will deploy all the Kiali resources (including the ConfigMap) for you - you do not install Kiali yourself. So it sounds like your operator is not managing your Kiali.

If I understood  this correctly, I would completely uninstall everything (Kiali, Operator, the CR), and then follow the getting-started guide at https://www.kiali.io/documentation/getting-started/

In short, you install the Operator. Once the operator is running and watching for CRs, then you create the CR for the operator to see. Once the operator sees the CR - it will install Kiali for you.

For a brief overview of how operators work, see: https://medium.com/@jmazzite/kubernetes-operators-a-very-brief-overview-270e75f3dfab

**Comment by zhaorendong** [comment]
Thanks, the problem has already been solved.
